app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionsTabFragment.java

```diff
@@ -137,15 +162,36 @@ public void onBindViewHolder(BindingHolder<ItemSessionBinding> holder, int posit
             ItemSessionBinding binding = holder.binding;
             binding.setSession(session);
 
+            boolean hideStartTime = false;
+
             if (position > 0 && position < getItemCount()) {
                 Session prevSession = getItem(position - 1);
-                if (prevSession.stime.getTime() == session.stime.getTime()) {
-                    binding.txtStime.setVisibility(View.INVISIBLE);
+
+                hideStartTime = prevSession.stime.getTime() == session.stime.getTime();
+            }
+
+            if (hideStartTime) {
+                binding.txtStime.setVisibility(View.INVISIBLE);
+            } else {
+                binding.txtStime.setVisibility(View.VISIBLE);
+            }
+
+            if (mode == MINE) {
+                boolean conflictWithNext = false;
+
+                if (position + 1 < getItemCount()) {
+                    Session nextSession =  getItem(position + 1);
+
+                    conflictWithNext = session.stime.getTime() == nextSession.stime.getTime();
+                }
+
+                if (hideStartTime || conflictWithNext) {
```

Comment:
In this case, `session` conflict with previous session if `hideStartTime` is `true`.
`hideStartTime` means that `session` and previous session have the same `stime` (start time).